Tasks/KubernetesManifestV0/task.json

```diff
@@ -0,0 +1,228 @@
+            "visibleRule": "connectionType = Azure Resource Manager",
+            "defaultValue": "",
+            "groupName": "kubernetesCluster",
+            "required": true
+        },
+        {
+            "name": "azureResourceGroup",
+            "label": "Resource group",
+            "type": "pickList",
+            "helpMarkDown": "Select an Azure resource group.",
+            "visibleRule": "connectionType = Azure Resource Manager",
+            "defaultValue": "",
+            "required": true,
+            "groupName": "kubernetesCluster",
+            "properties": {
+                "EditableOptions": "True"
+            }
+        },
+        {
+            "name": "kubernetesCluster",
+            "label": "Kubernetes cluster",
+            "type": "pickList",
+            "helpMarkDown": "Select an Azure managed cluster.",
+            "visibleRule": "connectionType = Azure Resource Manager",
+            "defaultValue": "",
+            "required": true,
+            "groupName": "kubernetesCluster",
+            "properties": {
+                "EditableOptions": "True"
+            }
+        },
+        {
+            "name": "namespace",
+            "type": "string",
+            "label": "Namespace",
+            "required": false,
+            "defaultValue": "",
+            "groupName": "kubernetesCluster",
+            "helpMarkDown": "Set the namespace for the kubectl command by using the –namespace flag. If the namespace is not provided, the commands will run in the default namespace."
+        },
+        {
+            "name": "action",
+            "type": "pickList",
+            "label": "Action",
+            "required": false,
+            "defaultValue": "deploy",
+            "groupName": "config",
+            "options": {
+                "deploy": "deploy",
+                "bake": "bake"
+            },
+            "helpMarkDown": "Manifests to deploy"
+        },
+        {
+            "name": "manifests",
+            "type": "filePath",
+            "label": "Manifests",
+            "required": false,
+            "defaultValue": "",
+            "groupName": "config",
+            "helpMarkDown": "Manifests to deploy",
+            "visibleRule": "action = deploy"
+        },
+        {
+            "name": "checkManifestStability",
+            "type": "boolean",
+            "label": "Check Manifests Stability",
+            "required": false,
+            "defaultValue": "",
+            "groupName": "config",
+            "helpMarkDown": "Waits on deployment status",
+            "visibleRule": "action = deploy"
+        },
+        {
```

Comment:
We should work with Shashank and Kalyan to understand what inputs it should be before releasing so that we can be forward compatible.

Reply:
I've removed `ARM Endpoint` as an option in the selector

Reply:
I'll talk to Shashank